docs/user/firststeps.rst

```diff
@@ -228,9 +235,78 @@ For example, if you want to keep the new notebook(s), run the following.
     git commit -m "Added some notebooks"
     git push
 
-Record you work and make it repeatable
+
+.. _interactive_exploration:
+
+Interactively explore the bicycle data
 """"""""""""""""""""""""""""""""""""""
 
+To start working with the bicycle data we have already created a sample
+notebook that does some data cleaning and visualization. We will first
+download the notebook so you can interactively explore the dataset, much like
+you would in a real project. Feel free to execute the cells. When we are ready
+to generate results, we will refactor the code from the notebook into a python
+module and run it with ``renku`` to create a repeatable analysis workflow.
+
+Use the commands below to add the notebook to your project.
+
+.. code-block:: console
+
+    mkdir -p notebooks
+    wget -O "notebooks/zhbikes-notebook.ipynb" https://raw.githubusercontent.com/SwissDataScienceCenter/renku/master/docs/_static/zhbikes/ZHBikes.ipynb
+    git add notebooks
+    git commit -m"Added zuerich bike notebook"
+    git push
+
+
+Refactor the notebook
+"""""""""""""""""""""
+
```

Comment:
Is it possible to say how you did this? Is there a special command for that? I can imagine it is useful for users to know!

Reply:
You mean refactoring? Did it by hand...

Reply:
Ah I thought maybe there was a secret python trick or so :]

Reply:
I remember @rokroskar complaining that copy/pasting from a notebook is not a thing

Reply:
Well it is a painful cell-by-cell thing